README.md

```diff
@@ -168,23 +178,33 @@ az pipelines variable-group variable create --group-id $SETT_GROUP_ID --secret t
 
 ```shell
 PIPELINE_NAME_DOCKER=jmeter-docker-build
+REPOSITORY_NAME=jmeter-load-test
+ORGANIZATION=https://dev.azure.com/your-organization
+PROJECT=your-project
 
 az pipelines create --name $PIPELINE_NAME_DOCKER --repository $REPOSITORY_NAME \
-    --repository-type tfsgit --branch master \
-    --yml-path pipelines/azure-pipelines.docker.yml
+                                                 --organization $ORGANIZATION \
+                                                 --project $PROJECT \
+                                                 --repository-type tfsgit --branch main \

```

Comment:
@robece loved the approach to set `main` branch as default with the `--repository-type tfsgit --branch main` command option. Thank you!

Reply:
Could you please remove the `--organization` and `--project` from the command?

Reply:
I fixed it on #45. Thanks @robece